functions/influxdb+grafana.bash

```diff
@@ -204,7 +204,7 @@ influxdb_install() {
   cond_echo ""
   echo "Installing InfluxDB..."
   dist="debian"
-  codename="stretch"
```

Comment:
This is a breaking change for users with a non-buster system willing to install influxdb

Reply:
This affects only the default value for codename; for Debian and Ubuntu it will be changed just a few lines later: codename="$(lsb_release -sc)". For other platforms either "stretch" or "buster" might not work...

Reply:
right of course. Carry on :)